crates/uv-auth/src/middleware.rs

```diff
@@ -1,6 +1,7 @@
 use std::sync::{Arc, LazyLock};
```

Comment:
Can you add cases for this new behavior to the unit tests in this module?

Reply:
Added